docs/release-notes/8.15.asciidoc

```diff
@@ -0,0 +1,70 @@
+
+[discrete]
+[[features-8.15.0]]
+==== New features
+
+* Introduces Automatic Import, a feature that helps you to quickly parse, ingest, and create ECS mappings for data from sources that don't yet have prebuilt Elastic integrations ({kibana-pull}186304[#186304]).
+* Adds the `scan` action to the response console, which allows you to scan a specific file or directory on a host for malware ({kibana-pull}184723[#184723]).
+* Adds an {elastic-defend} integration policy option in Advanced Settings that allows you to opt out of registry event filtering ({kibana-pull}186564[#186564]).
+* Allows you to specify additional file and registry paths to monitor for read access ({kibana-pull}181361[#181361]).
+* Allows you to use {elastic-sec} to isolate and release hosts running a CrowdStrike agent ({kibana-pull}186801[#186801]).
+* Allows you to retrieve files from SentinelOne-enrolled hosts ({kibana-pull}181162[#181162]).
+* Allows you to create an event filter that excludes the descendant events of a specific process ({kibana-pull}184947[#184947]).
+* Recalculates entity risk scores when asset criticality changes on an individual entity ({kibana-pull}182234[#182234]).
+* Adds an **Asset criticality** column to user and host data tables. If asset criticality levels are assigned to your users and hosts, this information appears in the **Asset criticality** column ({kibana-pull}186375[#186375], {kibana-pull}186456[#186456]).
+* Adds an API that allows you to perform paginated KQL searches through asset criticality records ({kibana-pull}186568[#186568]).
+* Adds public APIs for managing asset criticality ({kibana-pull}186169[#186169]).
+* Allows you to edit the `max_signals`, `related_integrations`, and `required_fields` fields for custom rules ({kibana-pull}179680[#179680], {kibana-pull}178295[#178295], {kibana-pull}180682[#180682]).
+* Provides help from AI Assistant when you're correcting rule query errors ({kibana-pull}179091[#179091]).  
+* Allows you to bulk update custom highlighted fields for rules ({kibana-pull}179312[#179312]).
+* Adds alert suppression for {ml} and {esql} rules ({kibana-pull}181926[#181926], {kibana-pull}180927[#180927]).
+* Provides previews of hosts, users, and alerts that you're examining in the alert details flyout ({kibana-pull}186850[#186850], {kibana-pull}186857[#186857]).
+* Adds an option for toggling row renderers on and off, and moves notes to a new flyout in Timeline ({kibana-pull}186948[#186948]).
+
+[discrete]
+[[enhancements-8.15.0]]
+==== Enhancements
+
+* Adds notifications to the connector dropdown menu on the **Attack discovery** page so you know when other connectors have new discoveries ({kibana-pull}186903[#186903], {kibana-pull}187209[#187209]).
+* Improves the output of discoveries generated with `GPT-4o` ({kibana-pull}187183[#187183]).
+* Allows Attack discovery generation to continue when you navigate to another page ({kibana-pull}184949[#184949]).
```

Comment:
You can also run discoveries on multiple connectors simultaneously. The attack discoveries are saved per user and they not only work across another page, but another browser. I would put this note ahead of the one that reads "Adds notifications to the connector dropdown menu on the Attack discovery..."

Reply:
Thanks for the review, @stephmilovic. Is there a PR tied to the functionality that you're describing? We normally add a link to the PR at the end of the change description so users can check it if they want to learn more.

Reply:
https://github.com/elastic/kibana/pull/184949